torchgen/dest/lazy_ir.py

```diff
@@ -406,26 +415,29 @@ def build_ir_node(self, func: NativeFunction, schema: LazyIrSchema) -> str:
         }}
         """
 
-    def create_lazy_tensor(self, first_tensor_name: str) -> str:
+    def create_lazy_tensor(self, first_tensor_name: Optional[str] = None) -> str:
         # xla uses an instance method for tensor creation, for the time being
         if self.create_from_first_tensor:
             # TODO(whc) remove this if XLA switches to using static method for creation
+            assert (
+                first_tensor_name is not None
+            ), "Requires first tensor to create lazy tensor"
             return f"{first_tensor_name}.{self.create_tensor}"
         return f"{self.backend_namespace}::{self.create_tensor}"
 
     def return_aten_tensor(self, func: NativeFunction, schema: LazyIrSchema) -> str:
         returns_length = len(schema.returns)
         value_args = schema.filtered_args(values=True, scalars=False)
         value_types_names = [f"{a.name}" for a in value_args if not a.is_wrapped_scalar]
-        assert (
-            len(value_types_names) > 0
-        ), "Code below assumes there is at least one tensor arg"
-        first_tensor_name = value_types_names[0]
+        first_tensor_name = value_types_names[0] if len(value_types_names) > 0 else None
         bridge_str = f"""auto result = {self.create_aten_from_ltc_tensor}(
                 {self.create_lazy_tensor(first_tensor_name)}(std::move(node), *common_device));"""
 
         if returns_length > 1:
-            bridge_str = f"""std::vector<{self.lazy_tensor_ptr}> lazy_tensors;
+            assert (
+                len(value_types_names) > 0
+            ), "Code below assumes there is at least one tensor arg"
+            bridge_str = f"""std::vector<{self.tensor_class}Ptr> lazy_tensors;
```

Comment:
Any specific reason to use `{self.tensor_class}Ptr` instead of `{self.lazy_tensor_ptr}`?

Reply:
The code used to have  {tensor_class}Ptr and I recenetly added the {lazy_tensor_ptr} field to allow @JackCaoG to customize it for XLA.  This might have just gotten lost during a rebase.  I'd agree it'd be better to change to `lazy_tensor_ptr`

Reply:
yea, xla need it to be `{self.tensor_class}Ptr` during the migration.

Reply:
wait, really? i thought you were setting `lazy_tensor_ptr` to be XlaTensorPtr or something, shouldn't both ways be equivalent for you?

Reply:
we currently do
```
        torch_xla::XLATensorPtr lazy_self = torch_xla::bridge::GetXlaTensorOrCreateForWrappedNumber(self, *common_device);
```

I need to make `XlaTensor` inherit from `LazyTensor` before I can use `LaztTensorPtr`

Reply:
and we already setting that today. @wconstab is correct, we don't need `{self.tensor_class}Ptr` anymore